Formula/colima.rb

```diff
@@ -37,6 +37,13 @@ def install
     (fish_completion/"colima.fish").write Utils.safe_popen_read(bin/"colima", "completion", "fish")
   end
 
+  service do
+    run [opt_bin/"colima", "start"]
```

Comment:
Almost works,

```
time="2022-01-31T15:09:35-06:00" level=info msg="using docker runtime"
time="2022-01-31T15:09:35-06:00" level=warning msg="error detecting Lima config directory: error retrieving lima info: exec: \"limactl\": executable file not found in $PATH"
time="2022-01-31T15:09:35-06:00" level=warning msg="falling back to default '$HOME/.lima'"
time="2022-01-31T15:09:35-06:00" level=fatal msg="Error: dependency check failed for VM: lima not found, run 'brew install lima' to install"
```

This is because `$PATH` doesn't contain `lima` and `limactl` which are required by colima since it calls it by shelling out. I've tried

```ruby
    environment_variables HOME: Dir.home(ENV["USER"]), PATH: "#{Formula['lima'].opt_bin}/:#{PATH}"
```

Which sets the PATH to what I'd expect, but it doesn't seem to find `limactl` regardless. Setting `HOME` as well so that it can find the user's config. Still digging.

Reply:
Try
```ruby
environment_variables HOME: ENV["HOME"], PATH: std_service_path_env
```

Reply:
That was perfect! It and one other change and it now works on my machine!